assets/js/components/user-input/UserInputQuestionWrapper.js

```diff
@@ -98,7 +98,7 @@ export default function UserInputQuestionWrapper( props ) {
 								<Button
 									className="googlesitekit-user-input__buttons--next"
 									onClick={ next }
-									disabled={ values.filter( ( value ) => value.trim().length > 0 ).length === 0 }
+									disabled={ values.some( ( value ) => value.trim().length === 0 ) || values.length === 0 }
```

Comment:
Generally speaking, we should always check the `length` (or other property) first before using a collection check to avoid unnecessary iterations. In this case, it doesn't really matter since there are no iterations to perform on an empty list but something to keep in mind as a general order of operations 👍